src/models/multiBarHorizontalChart.js

```diff
@@ -208,10 +209,19 @@ nv.models.multiBarHorizontalChart = function() {
 
                 g.select('.nv-x.nv-axis').call(xAxis);
 
-                var xTicks = g.select('.nv-x.nv-axis').selectAll('g');
+                var xTicks = g.select('.nv-x.nv-axis > g').selectAll('g');
 
                 xTicks
-                    .selectAll('line, text');
+                    .selectAll('line, text')
+                    .style('opacity', 1);
+
+                if (reduceXTicks)
+                    xTicks
+                        .filter(function(d,i) {
+                            return i % Math.ceil(data[0].values.length / (availableHeight / 24)) !== 0;
+                        })
+                        .selectAll('text, line')
+                        .style('opacity', 0);
             }
```

Comment:
So before it was showing a tick for every data point, and you want it to reduce that?